DrawerLayoutExample/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node_modules/react-native/packager/packager.sh"
+    "start": "react-native start"
   },
   "dependencies": {
-    "react-native": "^0.11.0",
-    "react-native-drawer-layout": "^0.1.0"
+    "react-native": "^0.17.0",
+    "react-native-drawer-layout": "file:.."
```

Comment:
Yes, as it makes it easier to test changes in the lib in this example application. (only dist/ and the package.json is copied, so the behavior should be the same)
Are there drawbacks I don't see or is it just unusual to do?